### ISSUE TITLE
Clearly separate autocomplete tags in lens dropdown

### ISSUE THREAD

**Issue by drolando** [openzipkin/zipkin#2485]
It's confusing why some tags would show up in the drop down and some wouldn't since right now everything's just mixed.

It'd be nice if we could split the list in 2 sections and move the autocomplete tags to the 2nd. With ideally a title that explains that only some whitelisted tags are there and they can use the generic "tags" search for everything else.

**Comment by drolando** [comment]
This is @tacigar's  comment on what'd be needed:

> I use react-select(https://github.com/jedwatson/react-select) to implement global-search component. (Please see https://github.com/openzipkin/zipkin/blob/7ac793e1261c4e03b1f1066cf9718c6c7c740d18/zipkin-lens/src/components/GlobalSearch/SearchCondition.js#L132-L169)
This component library is not very extensible.
So I think it is hard to add separator and descriptions to the dropdown as long as we use it.
It seems that we need to throw away react-select and create similar component ourself…
But it is a bit hard...

**Comment by codefromthecrypt** [comment]
I agree that for existing sites, they may be confused about which properties of the span are by default searchable. If we want to optimize for this, we could have a separate element for tags including predefined keys or blank to enter your own. Remember also that predefined should one day include site-specific tags that we don't auto-complete values for! ex antennae-id or request id.

on hover it could say how to control this.

**Comment by codefromthecrypt** [comment]
what if tags had a tag: or tag. prefix? not necessarily elegant but it
could reduce confusion perhaps. other data are top level

**Comment by codefromthecrypt** [comment]
I mean ex duration is really span.duration and name is span.name tag is
span.tags.foo

possibly namespacing could clarify, skipping the span. part

**Comment by codefromthecrypt** [comment]
I think haystack doesn't have this problem because you cannot search on and
hoc values iirc
